third-party/network_generator.py

```diff
@@ -119,4 +120,4 @@ def build(self):
 net.stop()
 
 # tests = { 'cli': cli }
-topos = {'dumbell': (lambda: DumbbellTopology())}
+topos = {'dumbell': (lambda: DumbbellTopology())}
```

Comment:
Missing newline at the end of file

https://thoughtbot.com/blog/no-newline-at-end-of-file